source/extensions/tracers/zipkin/util.h

```diff
@@ -49,13 +49,16 @@ class Util {
 
   /**
    * Returns a wrapped uint64_t value as a string. In addition to that, it also pushes back a
-   * replacement to the given replacements vector.
+   * replacement to the given replacements vector. The replacement includes the supplied name
+   * as a key, for identification in a JSON stream.
    *
    * @param value unt64_t number that will be represented in string.
+   * @param name std::string that is the key for the value being replaced.
    * @param replacements a container to hold the required replacements when serializing this value.
    * @return ProtobufWkt::Value wrapped uint64_t as a string.
    */
-  static ProtobufWkt::Value uint64Value(uint64_t value, Replacements& replacements);
+  static ProtobufWkt::Value uint64Value(uint64_t value, std::string name,
```

Comment:
when we have the name as constant, probably we can use `absl::string_view` as name here.

Reply:
Switched.